examples/00_quick_start/sasrec_amazon.ipynb

```diff
@@ -0,0 +1,912 @@
+{
```

Comment:
<p>This is a very nice function, it removes users and items with less than k interactions. @anargyri @simonzhaoms @angusrtaylor @aeroabir&nbsp;do you see us using this function in others part of the repo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1530/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Actually we have a function already <a href="https://github.com/microsoft/recommenders/blob/60033231b9167438032843c23158c0c776856e0e/recommenders/datasets/split_utils.py#L49" target="_blank">here</a>. You may use it instead. </p>

Reply:
@aeroabir if you want to do this change now, please let me know, otherwise we can create an issue and leave it for another PR